Wikipedia/Code/ArticleCollectionViewCell.swift

```diff
@@ -398,6 +398,11 @@ open class ArticleCollectionViewCell: CollectionViewCell, SwipeableCell, BatchEd
         }
     }
 
+    public override func layoutWidth(for size: CGSize) -> CGFloat {
```

Comment:
OK - I see now that it's for batch editing. This can move up into the superclass and we should ensure that interactive additions for swipe edits still work OK.